ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/polygon/DrawAreaTaskMapFragment.kt

```diff
@@ -57,7 +64,6 @@ class DrawAreaTaskMapFragment(private val viewModel: DrawAreaTaskViewModel) :
   }
 
   companion object {
-    fun newInstance(viewModel: DrawAreaTaskViewModel, map: MapFragment) =
-      DrawAreaTaskMapFragment(viewModel).apply { this.map = map }
+    const val TASK_ID_ARG_KEY = "taskId"
```

Comment:
AbstractTaskFragment already has this, how about using the same rather than creating this new?

Reply:
AbstractTaskFragment has the key used in `Bundle`, not Fragment args. They happen to have the same value, but that's coincidental rather than necessary.